src/py/reactpy/reactpy/core/hooks.py

```diff
@@ -204,7 +264,7 @@ def use_debug_value(
 
     if REACTPY_DEBUG_MODE.current and old.current != new:
         old.current = new
-        logger.debug(f"{current_hook().component} {new}")
+        logger.debug(f"{get_current_hook().component} {new}")
```

Comment:
Issue J - `current_hook` was renamed to `get_current_hook` to follow naming convention